tests/CacheCompat/CommonCache.Test.Unit/CommonCache.Test.Unit.csproj

```diff
@@ -10,8 +10,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
-    <PackageReference Include="MSTest.TestAdapter" Version="2.2.10" />
-    <PackageReference Include="MSTest.TestFramework" Version="2.2.10" />
+    <PackageReference Include="MSTest.TestAdapter" Version="3.0.2" />
```

Comment:
did we not move to use the central package management for this?

Reply:
Only for source projects and unit/integration tests, not cache compat and dev apps. We can add cache compat tests to CPM but it becomes a bit messy since each project references a specific ADAL or MSAL version.